### PR TITLE
Adds poquito

### PR DESCRIPTION
**Pull request by Ermucat**
## About the PR

Adds Poquito which is a new pet, but Im pretty sure everything is in the wrong files and it all has to be mapped ):
## Why / Balance

This would just add more ghost roles I guess? If mapped of course
## Technical details

Added Poquito to the list pets, added a spawner for poquito, added a preset for him, added an Id for him
## Media

https://github.com/user-attachments/assets/c430736b-3550-4cc6-aae9-9423eb72a40f

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

:cl:
- adds poquito the more mischievous version of pun pun

### CODE REVIEW

**Review by FluffMe — CHANGES_REQUESTED** [review]
* Yes, everything is in the wrong files.
* `Just added stuff from files` is not a very good technical description for me to go on. What stuff, from which files? Is this your code or attribution is needed?

**Comment by Ermucat** [comment]
Fair enough

File: Resources/Prototypes/Entities/Markers/Spawners/mobs.yml [hunk 1]
```diff
@@ -922,4 +922,4 @@
       sprite: Mobs/Animals/reindeer_doe.rsi
   - type: ConditionalSpawner
     prototypes:
-    - MobReindeerDoe
\ No newline at end of file
+    - MobReindeerDoe
```
Comment (FluffMe) [inline Resources/Prototypes/Entities/Markers/Spawners/mobs.yml:925]: Upstream file still shows as edited with only change being an EoF empty line added.
Please revert this change.

File: Resources/Prototypes/Roles/Jobs/Fun/misc_startinggear.yml [hunk 2]
```diff
@@ -102,6 +102,14 @@
...
     jumpsuit: ClothingUniformJumpsuitJacketMonkey
     id: PunPunIDCard
+# Harmony gear?
+- type: startingGear
+  id: MobPoquitoGear
+  equipment:
+    ears: ClothingHeadsetService
+    head: ClothingHeadHatChef
+    jumpsuit: ClothingUniformJumpsuitChef
+    id: PoquitoIDcard
```
Comment (FluffMe) [inline Resources/Prototypes/Roles/Jobs/Fun/misc_startinggear.yml:112]: * This prototype is already declared in `Resources/Prototypes/_Harmony/Roles/Jobs/Fun/misc_startinggear.yml`. Revert this change.

File: Resources/Prototypes/_Harmony/Entities/Objects/Misc/identification_cards.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+﻿# Harmony base ID card
```
Comment (FluffMe) [inline Resources/Prototypes/_Harmony/Entities/Objects/Misc/identification_cards.yml:1]: It's not a base card. Either remove the comment, or state that this is a Poquito ID card

File: Resources/Prototypes/_Harmony/Roles/Jobs/Fun/misc_startinggear.yml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+﻿# Harmony gear?
```
Comment (FluffMe) [inline Resources/Prototypes/_Harmony/Roles/Jobs/Fun/misc_startinggear.yml:1]: No question mark in the comment. `# Harmony Poquito gear` should be good

**Review by FluffMe — CHANGES_REQUESTED** [review]

**Comment by DieselMohawk** [comment]
I would like to note that in all maps would need to maintained in order to add the spawner to them. This may be a bit much, but I suppose it's up to Spanky in the end.

**Comment by FluffMe** [comment]
> I would like to note that in all maps would need to maintained in order to add the spawner to them. This may be a bit much, but I suppose it's up to Spanky in the end.

Admin spawn is also an option, ass spawner is available to them

File: Resources/Prototypes/Entities/Markers/Spawners/mobs.yml (hunk 1 above)
Comment (FluffMe) [inline Resources/Prototypes/Entities/Markers/Spawners/mobs.yml:925]: This change from previous review is still not addressed.

**Review by FluffMe — CHANGES_REQUESTED** [review]

**Comment by Ermucat** [comment]
As far as Im aware ( SO not much ) I would have to make a new species indentical to a kobold but instead of random sprite use a diffrent way to make his sprite consistent.

**Comment by FluffMe** [comment]
No, should be respritable by overriding stuff from base kobold in poquito prototype.

There are good spriters on Discord, if you need one to collaborate with. If memory doesn't fail me @Blackern5000 is a very good one too

**Comment by Ermucat** [comment]
thats why Im not the smartest. But the sprites are there its just setting a kobold color and the ears and tail

**Comment by Ermucat** [comment]
thanks for the help

File: Resources/Prototypes/Entities/Markers/Spawners/mobs.yml [hunk 5]
```diff
@@ -922,4 +922,5 @@
...
+
```
Comment (FluffMe) [inline Resources/Prototypes/Entities/Markers/Spawners/mobs.yml:926]: Still modified upstream file for no reason

**Review by FluffMe — CHANGES_REQUESTED** [review]
The change requested in previous 2 reviews is still not addressed.

**Pull request closed without merging**